main/lib/plugins/PALPIDEFSConverterPlugin.cc

```diff
@@ -1068,7 +1392,7 @@ namespace eudaq {
 
       for (unsigned int i_pixel = 0; i_pixel < n_pixels; ++i_pixel) {
         if (data[i_pixel * n_points] != 255) {
-          sector = i_pixel * 4 / 1024 / 512;
+          sector = i_pixel * 8 / 1024 / 512;
```

Comment:
Are you sure? If variable sector number is added, it would be easier also to adjust it for the next ALPIDE.